src/websocket.js

```diff
@@ -393,6 +393,12 @@ Strophe.Websocket.prototype = {
      * (Object) error - The websocket error.
      */
     _onError: function(error) {
+        // Workaround
+        // https://code.google.com/p/chromium/issues/detail?id=372776
+        if (this.socket.readyState === 0) {
+          return;
+        }
+
```

Comment:
@jcbrand hmm not it doesn't continues to send data frames to Strophe.js but it emits an error if data is received after the close frame has been sent (but shouldn't)